app/presenters/queries/embedded_content_presenter.rb

```diff
@@ -0,0 +1,43 @@
+module Presenters
+  module Queries
+    class EmbeddedContentPresenter
+      def self.present(target_edition_id, host_edition)
```

Comment:
Should the second parameter be `host_editions` (plural) or are we only passing one edition here? In the test we're passing multiple - should this be renamed, or am I misunderstanding here?

Reply:
Yes good spot! Thank you.